### ISSUE TITLE
Truffle Init - No Tests?

### ISSUE THREAD

**Issue by JimLynchCodes** [trufflesuite/truffle#4041]
Hi, just wondering why there are no tests in the scaffolded project when running `truffle init`...

Can I make a PR adding this? Where in the main truffle source code should I put this file?

thanks!

**Comment by cds-amal** [comment]
Hi @jimLynchCodes,

Thanks for bringing this issue up. By design there are no tests for truffle init but we could possibly include verbiage to guide developers on next steps, which would include tests.

cc: @kevinbluer , ideas?

**Comment by kevinbluer** [comment]
Hey @JimLynchCodes, 

One idea we had after a quick discussion would be to extend the output following a successful `init`...currently it's as follows:
```
Starting init...
================

> Copying project files to /Users/bluer/Developer/temp

Init successful, sweet!
```
This could potentially be elaborated upon to say something like: "_Why not try `truffle create contract HelloWorld` and `truffle create test HelloWorld` to create a new contract and its associated test?_"

PR very much welcome for this if you'd be interested. Location of the console output as it current stands is at the following: https://github.com/trufflesuite/truffle/blob/develop/packages/events/defaultSubscribers/init.js#L23

Thanks again 👍

**Comment by JimLynchCodes** [comment]
I like the "why not try `create contract` idea, but to me it seems weird to have to run an extra command to have it scaffold tests.

To me there isn't really a situation where you should want to scaffold a contract with no test (since, if you don't want it to work properly, why are you creating it in the first place? 😅)

**Comment by eggplantzzz** [comment]
Yes but after `truffle init` there are really no contracts to test yet.

**Comment by JimLynchCodes** [comment]
Hmm, good point. Maybe it makes sense to have an "Example" contract?

I have found that having only the Migrations.sol file that you are not supposed to touch is very confusing for beginners...

**Comment by Rishabhraghwendra18** [comment]
Hi @JimLynchCodes . First timer this side . is this issue open ?

**Comment by gnidan** [comment]
@Rishabhraghwendra18 looks like this issue should've been closed, assuming #4082 (now released) is sufficient in addressing this issue.

Closing this for issue maintenance, then, unless either of you report that there's more we could do. Thanks all!
